### ISSUE TITLE
Feature documentation?

### ISSUE THREAD

**Issue by Helveg** [BlueBrain/eFEL#208]
It's been a while that the eFeature documentation [here ](https://efel.readthedocs.io/en/latest/eFeatures.html) seems incomplete. Are there any plans of completing this guide? Currently I'm stuck trying to look through all of `efel.getFeatureNames()` to scout for interesting hits ;p Combine that with https://github.com/BlueBrain/eFEL/issues/27 and it's not so clear how exactly to use `efel` from Python when the feature isn't explained in the first link.

**Comment by DrTaDa** [comment]
Hello @Helveg, 
for the eFeature descriptions, all of the features should be documented in the pdf linked at the top of the page https://efel.readthedocs.io/en/latest/eFeatures.html. Did you notice any missing ?
For code snippets, it is true that the examples in the present repository are lacking, we could add new ones.

**Comment by Helveg** [comment]
Oh ok, there's an extra PDF? I'll look at that shortly but it would be nicer to have that in the sphinx doc ofcourse which is easier to search, navigate and index on search engines.

**Comment by wvangeit** [comment]
Yes, I know it's annoying to have a mix of documentation locations. As I mentioned in the other ticket, we're planning a big restructuring of the code. We will definitely make sure everything is correctly documented in one place there.
One of the reasons the pdf is still there is because there are quite some features that are never used. We also don't need to document all these never-used features.
If you let us know some important features you need, we can already now put them in the sphinx docs.

**Comment by Helveg** [comment]
Well some of the features seem to only exist for the first five spikes, is there a general n-th or all spike variant for them? Also, is frequency available as a feature?
